packages/app/src/cli/commands/app/deploy.ts

```diff
@@ -50,6 +50,11 @@ export default class Deploy extends Command {
         'Optional version tag that will be associated with this app version. If not provided, an auto-generated identifier will be generated for this app version.',
       env: 'SHOPIFY_FLAG_VERSION',
     }),
+    'source-control-url': Flags.string({
+      hidden: false,
+      description: 'The git reference url of the deployment.',
```

Comment:
```suggestion
      description: 'Optional url to associate this app version with a third-party source control version. This is for internal use only and won't be available externally.',
```

cc @jessehoek

Reply:
I'll take the message from https://github.com/Shopify/partners/blob/69317e0fdba91c803927a014cedf5688964854f0/app/graphql/graph_api/deployments/deployment_summary.rb#L17